src/software/gui/full_system/threaded_full_system_gui.h

```diff
@@ -7,12 +7,13 @@
 #include <thread>
 
 #include "shared/proto/tbots_software_msgs.pb.h"
-#include "software/ai/hl/stp/play_info.h"
+//#include "software/ai/hl/stp/play_info.h"
```

Comment:
why are we waiting to delete this line?

Reply:
That wasn't on purpose, good catch - fixed